src/api/css.ts

```
@@ -185,6 +185,78 @@ function stringify(obj: Record<string, string>): string {
+      // Skip the escaped character.
+      i += 1;
+      continue;
+    }
+
+    if (quote) {
+      if (char === quote) quote = null;
+      continue;
+    }
+
+    // A comment can hold an unbalanced quote or a semicolon, so it has to be
+    // skipped before either is interpreted.
```

ESLint

[error] Expected a block comment instead of consecutive line comments.

(multiline-comment-style)